Assets/Scripts/UserInterface/GameUI/LevelUpRewardScreenController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using UnityEngine;
+using UnityEngine.UI; 
+using System.Collections;
+
+public class LevelUpRewardScreenController : UIController {
+
+    public GameObject newCanvas;
+    public Text text;
+    public string textToDisplay;
+    public int secondsToWait = 5;
+
+    public static bool Leveled;    
+
+    void Start()
+    {
+        newCanvas.SetActive(false);        
+        Leveled = false;
+    }
+
+    void Update()
```

Comment:
So this is the one thing I'd still like to change. Ideally we're not using the "Update" method for anything that doesn't require constant changes (e.g. animation). There are definitely ways you can accomplish this same functionality without checking in update, e.g. using an EventController call or by making a `public static LevelUpRewardScreenController Instance;` var that you reference in the other script